bowling/player.cpp

```diff
@@ -0,0 +1,78 @@
+#include "player.hpp"
+
+#include <iomanip>
+#include <iostream>
+#include <iterator>
+#include <sstream>
+
+Player::Player() {}
+
+Player::Player(std::string name, std::vector<std::pair<int, int>> points)
+    : name_(name),
+      points_(points) {}
+
+std::string Player::getName() const {
+    return name_;
+}
+
+int Player::getScore() const {
+    return score_;
+}
+
+void Player::setScore(int score) {
+    score_ = score;
+}
+
+void Player::countScore(std::vector<std::pair<int, int>> points) {

```

Comment:
well, I would rather expect that it's not pair of int and int, but char and char. In input, we can have values like '1', 'X', '/', '-'
Unless you have some processor that already make this translation.

Reply:
yes we've one it in ```std::string Player::translateVectorToString()```